### PR TITLE
HV-1021 Cache result of loadClass for el dependencies checking

### PR DESCRIPTION
**Pull request by lucaspouzac**

### CODE REVIEW

**Comment by hferentschik** [comment]
Hi @lucaspouzac, in order to accept your patch, could you sign  the JBoss CLA - https://cla.jboss.org/index.seam? All you need is a free JBoss.org user account and a couple of clicks.

**Comment by lucaspouzac** [comment]
It's done

**Comment by hferentschik** [comment]
@lucaspouzac, thanks for the pull request and your interest in Validator. Pull request merged.

File: engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
 
 	private static final Log log = LoggerFactory.make();
 
+	private static Boolean missingElDependencies;
```
Comment (gunnarmorling) [inline engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java]: Should this not better be volatile to safely publish the write to other threads?

**Pull request closed without merging**